app/resources/admin_acls.html.template

```diff
@@ -16,15 +16,16 @@
 {% block content %}
   <h2>Grant or revoke admin access</h2>
 
-  <h3>Add new admin or moderator</h3>
+  <h3>Add new admin</h3>
 
   <form method="post">
     <input type="hidden" name="xsrf_token" value="{{xsrf_token}}" />
     <p>
       Email address: <input type="text" name="email_address" /><br/>
       Access level:
       <select name="level">
-        <option value="administrator">Administrator</option>
+        <option value="superadmin">Superadmin</option>
+	<option value="manager">Manager</option>
```

Comment:
(most insightful comment of the year)
maybe add spaces to line it with the other <option>s?